qhub/cli/__init__.py

```diff
@@ -51,6 +51,9 @@ def cli(args):
     if QHUB_GH_BRANCH:
         logging.info(f"Modifying for development branch {QHUB_GH_BRANCH}")
 
+    if QHUB_IMAGE_TAG:
+        logging.info(f"Modifying image tag, using `{QHUB_IMAGE_TAG}` instead.")
```

Comment:
Modifying what to image, tag? Unless we are explicitly setting some config then overwriting this logging message and the above one seem particularly cryptic.

Reply:
I updated these messages to be a bit more informative and moved them to more appropriate places 👍